src/cards/top-languages-card.js

```diff
@@ -734,6 +734,8 @@ const renderTopLanguages = (topLangs, options = {}) => {
     layout,
     custom_title,
     locale,
+    card_width = DEFAULT_CARD_WIDTH,
```

Comment:
You don't need to change the top-languages-card.js file for your pull request.

Reply:
I just make it to default since previously I changed  it